build.gradle

```diff
@@ -29,15 +29,29 @@ repositories {
     jcenter()
 }
 
+// See https://docs.gradle.org/2.6/userguide/test_kit.html
```

Comment:
I'd prefer if we use nebula or Gradle 2.13's built in stuff rather than doing this manually.